.github/workflows/eden_gcp.yml

```diff
@@ -164,8 +164,10 @@ jobs:
           if [ "${{ matrix.backend }}" == "gcp" ]; then
             ./eden utils gcp vm log --vm-name eve-edengcp-actions-${{ matrix.hv }}-${{ matrix.fs }}-${{github.run_number}} -k "${{ steps.gcpauth.outputs.credentials_file_path }}" > console.log || echo "no device log"
           else
-            ./eden rol rent console-output -p "$ROL_PROJECT" -i "$rent_id" > console.log || echo "no device log"
+            ./eden rol rent console-output -p "$ROL_PROJECT" -i "${{ steps.rol.outputs.id }}" > console.log || echo "no device log"
           fi
+        env:
+          ROL_PROJECT: ${{ secrets.ROL_PROJECT }}
```

Comment:
Ah, when you mentioned the problem, I can see it, thank you for fixing. Probably it would be better to integrate ROL workflow into Eden project to check such problems before upstreaming, they are quite difficult to predict.
BTW, do we need `ROL_API_URL` and `ROL_API_KEY` here as well?

Reply:
Yes, we need it, thanks.

We can add RoL integration to eden workflow tests, but we only have 10 RPi4s in the pool and EVE workflow uses 4 RPi4s at a time, if we add multiple tests to eden, RoL can't allocate that many Rpi4s for tests.

For the current RoL, we can only run two workflow tests at the same time.

Reply:
Got it. What do you think about limiting of concurrent runs of EdenGCP workflows: https://github.com/lf-edge/eve/pull/2739? Using this approach we will run on 4 RPI4 at a time instead of unpredictable number (up to 20 for free github account).